data/keywords/tei/k000013.xml

```diff
@@ -11,27 +11,30 @@
             <title level="m">Architectura Sinica</title>
             <principal xml:id="TGM">Tracy Miller</principal>
             <editor role="general" ref="#TGM">Tracy Miller</editor>
+            <editor role="associate" ref="#TGM">Tracy Miller</editor>
             <editor role="content-author" xml:id="BY">BAI Ying 白穎</editor>
             <editor role="content-author" xml:id="CXT">CHEN Xintao 陳欣濤</editor>
-            <editor role="code-author" ref="#TGM">Tracy Miller</editor>
             <respStmt>
-               <resp>Initial research and review by </resp>
+               <resp>Initial research and revision by</resp>
                <name type="person" ref="#BY">BAI Ying 白穎, </name>
-               <name type="person" ref="#CXT">CHEN Xintao 陳欣濤, </name>
+               <name type="person" ref="#CXT">CHEN Xintao 陳欣濤</name>
+            </respStmt>
+            <respStmt>
+               <resp>Peer review by</resp>
                <name type="team">ATTCAT 2018</name>
             </respStmt>
             <respStmt>
-               <resp>Editing by </resp>
-               <name type="person" ref="#TGM">Tracy Miller, </name>
-               <name type="person" xml:id="AC">Aurelia Campbell</name>
+               <resp>Editing and proofreading by</resp>
+               <name type="person" ref="#TGM">Tracy Miller </name>
```

Comment:
Zane, 
do not remove Aurelia's name. 
in this case, use Aurelia and remove Tracy's name